sdk/formrecognizer/azure-ai-formrecognizer/README.md

```diff
@@ -295,9 +307,9 @@ for (int i = 0; i < receiptPageResults.size(); i++) {
 Train a machine-learned model on your own form type. The resulting model will be able to recognize values from the types of forms it was trained on.
 Provide a container SAS url to your Azure Storage Blob container where you're storing the training documents. See details on setting this up
 in the [service quickstart documentation][quickstart_training].
-<!-- embedme ./src/samples/java/com/azure/ai/formrecognizer/ReadmeSamples.java#L170-L190 -->
+<!-- embedme ./src/samples/java/com/azure/ai/formrecognizer/ReadmeSamples.java#L195-L215 -->
 ```java
-String trainingFilesUrl = "{training_set_SAS_URL}";
+String trainingFilesUrl = "{<SAS-URL-of-your-form-folder-in-blob-storage>}";
```

Comment:
"SAS-URL-of-your-form-folder-" makes me think I need to include the folder name in the url. I think "SAS-URL-of-your-container-in-blob-storage" might make more sense.